BraintreeCore/src/main/java/com/braintreepayments/api/Configuration.kt

```diff
@@ -354,9 +351,9 @@ open class Configuration internal constructor(configurationString: String?) {
         isBraintreeApiEnabled = braintreeApiConfiguration.isEnabled
         isFraudDataCollectionEnabled = cardConfiguration.isFraudDataCollectionEnabled
         isGraphQLEnabled = graphQLConfiguration.isEnabled
-        isKountEnabled = kountConfiguration.isEnabled
+        isKountEnabled = false
```

Comment:
Do we need `isKountEnabled` or `kountMerchantId` at all anymore?

Reply:
Cool - maybe we can add a `NEXT_MAJOR_VERSION` note here so we don't forget to remove it in v5